src/redux/walletconnect.js

```diff
@@ -278,6 +277,7 @@ export const walletConnectOnSessionRequest = (uri, callback) => async (
         // We need to add a timeout in case the bridge is down
         // to explain the user what's happening
         timeout = setTimeout(() => {
+          meta = android ? Navigation.getActiveRoute().params.meta : meta;
```

Comment:
any reason this one doesn't have the optional chaining but the other one (line 291) does?

Reply:
+1

Reply:
none really